library/Zend/Code/Reflection/ClassReflection.php

```diff
@@ -230,11 +230,17 @@ public function getProperties($filter = -1)
         return $zendReflections;
     }
 
+    /**
+     * @return string
+     */
     public function toString()
     {
         return parent::__toString();
     }
 
+    /**
+     * @return string
+     */
     public function __toString()
```

Comment:
If this method only calls `parent::__toString()`, it has no use and can be removed.